### PR TITLE
cleaning up docs in lightly/data subpackage

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
Hey there, 

For now i have updated the docs for the helper functions only, and needed some review before i finish up with rest. Do let me know if you would like some modifications. I will finish up with rest very soon. Thanks :)

### CODE REVIEW

**Comment by ChiragAgg5k** [comment]
Screenshot 2024-10-09 at 11 56 00 PM

i forgot to run the pre-commit, but now that i did, seems like everyone else also forgot to do so?? 😅

File: lightly/data/_helpers.py [hunk 1]
```diff
@@ -34,104 +34,102 @@
...
 
     Returns:
-        Dataset consisting of images/videos in the root directory.
+        A dataset object (VideoDataset, ImageFolder, or DatasetFolder) based on the directory contents.
 
     Raises:
-        ValueError: If the specified dataset doesn't exist
-
+        ValueError: If the specified dataset directory doesn't exist or if videos are present
+                    but VideoDataset is not available.
```
Comment (guarin) [inline lightly/data/_helpers.py:116]:
```suggestion
        ValueError: If the specified dataset directory doesn't exist or if videos are present
            but VideoDataset is not available.
```

We usually don't align indents but instead just indent one level more.

**Review by guarin — CHANGES_REQUESTED** [review]
Thanks a lot for the PR, this looks very good! Just one small comment but otherwise this is ready to be merged.

Interesting that the pre-commit hooks try to format the `openapi_generated` files, those should be left untouched.

**Comment by guarin** [comment]
The Code Format Check action is failing, could you run `make format` and update the PR?

**Review by guarin — APPROVED** [review]
LGTM thanks so much!

**Pull request merged**